GameOfLife/Board.php

```diff
@@ -176,4 +184,28 @@ function printBoard()
             echo "\n";
         }
     }
+
+    /**
+     * @meth
+     *
+     *Checks if the generations repeat
```

Comment:
Please fix the documentation!
- `@meth` does not mean anything.
- Add a space before Check and make it the first line of the comment and also ensure that it ends with a .
- Also improve the sentence because it is not entirely accurate.